.gitignore

```diff
@@ -24,3 +24,5 @@ package-lock.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+yarn.lock*
```

Comment:
This absolutely needs to remain. Remove this immediately or else Docusaurus will fail to run for everyone else. Revert this. You need to upgrade to the latest version of Docusaurus.